docs/core-protocol/smart-contracts-api/PublicLock.md

```diff
@@ -1192,22 +1188,8 @@ Allows a Lock manager to add or remove an event hook
 | _onValidKeyHook | address | Hook called to determine if the contract should overide the status for a given address |
 | _onTokenURIHook | address | Hook called to generate a data URI used for NFT metadata |
 | _onKeyTransferHook | address | Hook called when a key is transfered |
-
-### setExpirationDuration
-
-```solidity
-function setExpirationDuration(uint256 _newExpirationDuration) external nonpayable
```

Comment:
Are we really removing this function?

Reply:
Yes! replaced with a single setter for multiple things at once in order to save bytecode size...